### DISCUSSION TITLE
33rd open maintainers call

### DISCUSSION THREAD

**Discussion by mxinden** [libp2p/rust-libp2p#4977] [General]
Hi everyone,

The rust-libp2p maintainers will meet tomorrow (Tuesday) 11:00 UTC. See [libp2p public events calendar](https://calendar.google.com/calendar/embed?src=libp2p.io_0q9682i3te7eanhe9q7ae1c58g%40group.calendar.google.com).

https://meet.jit.si/rust-libp2p-open-maintainers-call

Everyone is welcome to join, listen along, propose topics and join the discussions.

Please post below in case you already want to add an agenda item or list it in [our notes document](https://hackmd.io/xoHKlFuER0OVj1DGDuFROw).

See you all tomorrow!

**Comment by thomaseizinger** [comment]
I might be able to make this one, depends on how long I am gonna stay at the Rust meetup! Don't wait for me though :)
